types/mongoose/index.d.ts

```diff
@@ -200,51 +201,36 @@ declare module "mongoose" {
      */
     constructor(base: typeof mongoose);
 
-    /**
-     * Opens the connection to MongoDB.
-     * @deprecated open() is deprecated in mongoose >= 4.11.0
-     * @param mongodb://uri or the host to which you are connecting
```

Comment:
Why breaking compatibility with older version of mongoose ?

Reply:
@emmanuelgautier the `open` method is no longer present in mongoose 5.5. I didn't trace back to see when they removed it, my guess would be 5.0, but it most definitely is not there now. As such it is incorrect to have the types indicate that the method is still available.

Reply:
It was me who removed open() and openSet() api docs from mongoose. @jloveridge is right.
Mongoose made this breaking changes at `5.0.0-rc0 / 2017-12-28`